clang/unittests/ASTMatchers/ASTMatchersNodeTest.cpp

```diff
@@ -556,6 +556,22 @@ TEST_P(ASTMatchersTest, DeclRefExpr) {
                          Reference));
 }
 
+TEST_P(ASTMatchersTest, DependentScopeDeclRefExpr) {
+  if (!GetParam().isCXX() || GetParam().hasDelayedTemplateParsing()) {
+    // FIXME: Add a test for `dependentScopeDeclRefExpr()` that does not depend
```

Comment:
The concept of template-dependence (and thus `DependentScopeDeclRefExpr` nodes) only exist in C++, so it's expected that the test depends on C++.

Based on some other tests that check this condition, please revise the comment to instead say:

```
// FIXME: Fix this test to work with delayed template parsing.
```

Reply:
Thank you, I updated it :D